docker/client.Dockerfile

```diff
@@ -20,7 +20,6 @@ WORKDIR /app
 COPY nightfall-client/src src
 COPY nightfall-client/docker-entrypoint.sh nightfall-client/package.json nightfall-client/package-lock.json ./
 
-RUN npm link @polygon-nightfall/common-files
```

Comment:
Just leaving a comment to remind us of this change (since we are modifying this from different PRs)